geth/api/api.go

```diff
@@ -168,9 +168,8 @@ func (api *StatusAPI) JailParse(chatID string, js string) string {
 }
 
 // JailCall executes given JavaScript function w/i a jail cell context identified by the chatID.
-// Jail cell is clonned before call is executed i.e. all calls execute w/i their own contexts.
-func (api *StatusAPI) JailCall(chatID string, path string, args string) string {
-	return api.b.jailManager.Call(chatID, path, args)
+func (api *StatusAPI) JailCall(chatID, this, args string) string {
+	return api.b.jailManager.Call(chatID, this, args)
```

Comment:
Can you give more context on `path`->`this` rename? `Path` have some meaning, but `this` is quite ambiguous. I don't see `path` as a good name though. Maybe just `name` or `function`?

Reply:
Check here: https://github.com/status-im/status-go/blob/5fb4aef1cc6866655641ba28c96ddfef7fff2e87/vendor/github.com/robertkrimen/otto/otto.go#L536

Reply:
Ah, ok. Makes sense now, thanks!